client/components/button/test/index.js

```diff
@@ -0,0 +1,1064 @@
+/**
+ * Created by nitrajka on 09/10/16.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../';
+import Gridicon from 'components/gridicon';
+
+describe( 'Button', function() {
```

Comment:
With ES6, you could use arrow functions here and throughout as a more concise alternative to the anonymous function, especially when you don't need to access `this`, which is generally true in our tests.

```js
describe( 'Button', () => {
```